client/layout/style.scss

```diff
@@ -336,3 +336,36 @@
 		}
 	}
 }
+
+
+// The following is for moving cart contents to the right.
+// As implemented in the showCheckoutCartRight ab test.
+@include breakpoint( '>660px' ) {
+	div.checkout__container {
+		display: flex;
+	}
+
+	div.checkout__container .main {
+		margin-right: 0;
+		width: 100%;
+	}
+
+	div.checkout__container .secondary-cart {
```

Comment:
Let's wrap the `<ul>` element with a `<div>` so the right column doesn't run super long without reason. See screenshot for the final look and also for the CSS I've applied (all changes are done via element style, so they're visible there).

![image](https://user-images.githubusercontent.com/538790/57693786-5fe4d300-764a-11e9-9e7e-0dc8dc45e90a.png)

Here's a  rundown of the changes applied: 

- Adding the extra `div` around the `ul` takes care of the long right column. Margins need to be used to keep the centering of both elements.
- That lets us apply `box-shadow` to the `ul` column (so it matches the main column).
- The rest of the changes goes to `cart__header` to address the weird borders. We need to remove `box-shadow` and `border-top`, and add `border-bottom`. Also `line-height` is adjusted to match the card header style of the main column.

Let me know if questions!

Reply:
These changes are done in [d566f03](https://github.com/Automattic/wp-calypso/pull/32832/commits/d566f0331ccb70d927297fdb648ddbd8dfdadf38) and [a650be6](https://github.com/Automattic/wp-calypso/pull/32832/commits/a650be63b3bf15a8a39db5d4b30086fbf7c7efb9).

> Adding the extra div around the ul takes care of the long right column. Margins need to be used to keep the centering of both elements.

A simpler fix is to use the `align-self` property, which I've implemented.

> Also line-height is adjusted to match the card header style of the main column.

Is the `line-height` adjustment to align the border-bottom of the headers in the `main` and `secondary-cart` column? If yes, then `line-height` alone will not help since the header in the main column is different in other views (check the header in the domain contact form for example). I've used `padding` to achieve the border alignments in the two headers.

Reply:
Awesome! Great improvements :clap: